Mathlib/Data/Nat/Digits.lean

```diff
@@ -516,6 +517,59 @@ lemma self_div_pow_eq_ofDigits_drop (i n : ℕ) (h : 2 ≤ p):
     (fun l hl ↦ digits_lt_base h hl)
   exact (ofDigits_digits p n).symm
 
+open BigOperators
```

Comment:
If you also open `Finset` then you'll be able to drop a bunch of `Finset.` prefixes below.
```suggestion
open BigOperators Finset
```